nemo/collections/asr/data/audio_to_text.py

```diff
@@ -46,24 +46,30 @@ def _speech_collate_fn(batch, pad_id):
     """
     _, audio_lengths, _, tokens_lengths = zip(*batch)
     max_audio_len = 0
-    max_audio_len = max(audio_lengths).item()
+    has_audio = audio_lengths[0] is not None
+    if has_audio:
+        max_audio_len = max(audio_lengths).item()
```

Comment:
regarding `has_audio`. Please remove this from `_speech_collate_fn` and add any needed